android/app/build.gradle

```diff
@@ -40,7 +40,7 @@ android {
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
         applicationId "com.example.knowyourdonor"
-        minSdkVersion 16
```

Comment:
Did you try running the app after making this change ?

Reply:
their are some spelling mistakes in the main flutter code..should i also correct them???

Reply:
You can make separate issue for this and send a separate PR for that also.

Reply:
this app is  running fine

Reply:
Cool! I have just added the static UI for the app.

Reply:
Why my pull request is not still not accepted?

Reply:
Why my pull request is not still accepted?

Reply:
You need to write meaningful commit messages. You can have a look at CONTRIBUTING.MD for that.